client/src/components/info/restaurants.js

```diff
@@ -0,0 +1,47 @@
+const sortBy = ['Relevance', 'Distance', 'Your match'];
+const city = ['Вінницька область', 'Волинська область', 'Дніпропетровська область', 'Донецька область',
```

Comment:
Don't use non Latin letters in code. If you need localization, use special tools for it